CONTRIBUTING.md

```diff
@@ -105,7 +105,7 @@ While the prerequisites above must be satisfied prior to having your pull reques
 
 ### PHP Styleguide
 
-All JavaScript must adhere to [PHP The Right Way](https://phptherightway.com/#code_style_guide) code style.
+All JavaScript must adhere to [PHP The Right Way](https://phptherightway.com/#code_style_guide) code style.
```

Comment:
Now I know that you miss JavaScript <3

Reply:
It was a non-breakable space, that I happen to type sometimes on my Bépo keyboard :-)

Reply:
> It was a non-breakable space

:sweat_smile: in fact didn't notice that

I was talking about the fact you are speaking about Javascript in the PHP Styleguide section and point to a PHP coding style. Or maybe there are rules about JavaScript in this styleguide I didn't know about?!


All JavaScript must adhere to [PHP The Right Way](https://phptherightway.com/#code_style_guide) code style.	All JavaScript must adhere to [PHP The Right Way]

Reply:
LOL ! I didn’t even notice my mistake, despite your comment !